src/main/java/model/StepExistenceGenerator.java

```diff
@@ -1,11 +1,25 @@
 package model;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Random;
 
 public class StepExistenceGenerator {
     private static final Random random = new Random();
 
-    public static boolean generate(boolean priorExistence) {
+    public static List<Step> generate(int numberOfParticipants) {
+        List<Step> steps = new ArrayList<>();
+        boolean priorExistence = false;
+
+        for (int i = 1; i < numberOfParticipants; i++) {
+            boolean doesExist = generateEachExistence(priorExistence);
+            priorExistence = doesExist;
+            steps.add(Step.findByExistence(doesExist));
+        }
```

Comment:
```suggestion
        for (int i = 1; i < numberOfParticipants; i++) {
            boolean doesExist = generateEachExistence(priorExistence);
            steps.add(Step.findByExistence(doesExist));
            priorExistence = doesExist;
        }
```

[의견] 이 부분은 정말 사소한 의견이고 개인적인 의견이여서 참고만 해주세요 :) 제가 느끼기에 로직의 흐름이 위 코드처럼 나열되는게 더 자연스러운 것 같아서 제안드려봅니다. 

코드의 맥락은 '이전 스탭에 따라서 사다리의 생성이 결정된다. 따라서 이전 스탭을 확인하면서 새로운 스탭을 생성하고, 이후 다음 스탭 생성을 위해 지금의 스탭으로 prev를 초기화한다'라는 맥락이라고 생각합니다.

그렇다면 이번 단계에서 생성한 스탭을 먼저 추가한 후 다음 스탭을 생성하기 위한 세팅이 진행되는게 더 자연스러울 것 같아요. 정말 사소한 부분일 수 있지만 제가 느끼기엔 약간의 이질감이 느껴졌던 것 같은데 엘라가 한번 읽어보고 공감이 간다면 반영해주셔도 좋을 것 같아요 👍🏼

Reply:
같은 기능이지만 코드 순서를 어떻게 배치하는지에 따라 읽히는 느낌이 달라지네요! Step에 추가하고나서 추가된 값을 priorExistence에 초기화하는게 흐름적으로 더 적절한것같습니다. 해당 부분 수정하였습니다! 감사합니다.👍🏻👍🏻